components/LoginForm.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import {
+  Box,
+  Text,
+  Heading,
+  VStack,
+  FormControl,
+  Input,
+  Button,
+  Center,
+  Image,
+  Alert,
+  HStack,
+  IconButton,
+  CloseIcon,
+  Collapse,
+  ScrollView,
+} from "native-base";
+import { StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
+  container: {
+    marginTop: 20,
+    textAlign: "center",
+  },
+});
+
+function LoginForm() {
+  const [show, setShow] = React.useState(false);
+  const WrongInformationGivenError = (
+    <Collapse isOpen={show} duration={20}>
+      <Alert
+        w="285"
+        status="error"
+        marginTop="5"
+        onTouchEndCapture={() => setShow(false)}
+      >
+        <VStack space={1} flexShrink={1} w="100%">
+          <HStack
+            flexShrink={1}
+            space={2}
+            alignItems="center"
+            justifyContent="space-between"
+          >
+            <HStack space={2} flexShrink={1} alignItems="center">
+              <Alert.Icon />
+              <Text fontSize="md" color="coolGray.800">
+                Feil brukernavn eller passord
+              </Text>
+            </HStack>
+            <IconButton
+              variant="unstyled"
+              _focus={{
+                borderWidth: 0,
+              }}
+              icon={<CloseIcon size="3" />}
+              _icon={{
+                color: "coolGray.600",
+              }}
+            />
+          </HStack>
+        </VStack>
+      </Alert>
+    </Collapse>
+  );
+  return (
+    <ScrollView w="100%" h="80">
+      <Center w="100%">
+        <Box safeArea p="2" py="8" w="90%" maxW="290">
+          <Image
+            source={{
+              uri: "https://lego.abakus.no/static/assets/img/logo.png",
+            }}
```

Comment:
It's probably smarter to use local assets instead of loading from the interweb :sweat_smile:

Reply:
Agreed 👍